### ISSUE TITLE
tensorboard doesn't show learning rate 

### ISSUE THREAD

**Issue by coder1248** [tensorflow/tensor2tensor#1777]
After training with evolved_transformer on a TPU (with evolved big tpu hparams), when starting tensorboard I only see the loss and steps, nothing else.  Actually it happens also with the regular transformer.
Specifically I want to see the learning rate too.

Can anyone help me find where does the logging to tensorboard happen in the code? I can't find it. I want to check the logging of the loss (which works) against the logging of the learning rate (which doesn't), so I may have a chance to fix it somehow or try something else.

Thanks! 

...

### Environment information

```
OS: Debian 9

$ pip freeze | grep tensor
mesh-tensorflow==0.0.5
tensor2tensor==1.13.4
tensorboard==1.12.0
tensorflow==1.13.1
tensorflow-datasets==1.0.2
tensorflow-estimator==1.13.0
tensorflow-hub==0.4.0
tensorflow-metadata==0.13.0
tensorflow-probability==0.6.0
tensorflow-serving-api==1.12.0

$ python -V
Python 2.7.13

*At the beginning of training, I get this warning:
```
# Error logs:
...
WARNING:tensorflow:56 unsupported operations found: 
  ScalarSummary (evolved_transformer/input_stats/batch_prediction_key_batch)
  ScalarSummary (evolved_transformer/input_stats/batch_prediction_key_length)
  ScalarSummary (evolved_transformer/input_stats/batch_prediction_key_nonpadding_tokens)
  ScalarSummary (evolved_transformer/input_stats/batch_prediction_key_nonpadding_fraction)
  ScalarSummary (evolved_transformer/input_stats/inputs_batch)
WARNING:tensorflow:... and 51 more
```
